tests/test_choseninlineresult.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2016
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+
+"""This module contains a object that represents Tests for Telegram
+ChosenInlineResult"""
+
+import os
+import unittest
+import sys
+sys.path.append('.')
+
+import telegram
+from tests.base import BaseTest
+
+
+class ChosenInlineResultTest(BaseTest, unittest.TestCase):
+    """This object represents Tests for Telegram ChosenInlineResult."""
+
+    def setUp(self):
+
+        user = telegram.User(1, 'First name')
+
+        self.result_id = 'result id'
+        self.from_user = user
+        self.query = 'query text'
+
+        self.json_dict = {
+            'result_id': self.result_id,
+            'from': self.from_user.to_dict(),
+            'query': self.query
+        }
+
+    def test_choseninlineresult_de_json(self):
+        """Test ChosenInlineResult.de_json() method"""
+        print('Testing ChosenInlineResult.de_json()')
```

Comment:
you don't really need the docstring and print here.
when running `nosetests -v` it will print the method name by default and imho this is the most informative for the developer.